docs/freebsd/README.md

```diff
@@ -47,7 +47,10 @@ Then, copy out the binary back to host into `bin/freebsd_amd64` dir.
 ## Setting up the FreeBSD VM
 
 It is easiest to start with a [snapshot image](http://ftp.freebsd.org/pub/FreeBSD/snapshots/VM-IMAGES/13.0-CURRENT/amd64/Latest/) of FreeBSD.  Fetch a QCOW2 disk image for QEMU or a raw image for GCE or bhyve.
-
+Fetch a copy of the FreeBSD kernel sources and place them in `/usr/src`. You will likely need to expand the disk of the VM. Before booting the VM, run:
+```console
+# truncate -s 15G $IMAGEFILE
+```
 To enable KCOV on FreeBSD, a custom kernel must be compiled.  It is easiest to do this in the VM itself.  Use QEMU to start a VM using the downloaded image:
```

Comment:
These two sentences are wrong now, we are compiling the kernel on the host.

The order of the steps is also a bit confusing. Here we are booting the VM to configure a few settings in loader.conf, then building a kernel on the host and installing it, and then booting the VM again to enable sshd. We can build and install the kernel first, then boot the VM once to do all needed configuration.